httputil/json.go

```diff
@@ -25,3 +25,12 @@ func WriteJSONResponse(w http.ResponseWriter, code int, resp interface{}) error
 	}
 	return nil
 }
+
+func WriteError(w http.ResponseWriter, code int, msg string) error {
```

Comment:
I agree that different API's will likely have their own error formats. I think it's nice to have a wrapper though. It would be easy enough to define in other repos if we really wanted though